editor/src/core/property-controls/property-controls-processor.spec.ts

```diff
@@ -0,0 +1,110 @@
+import { PropertyControlsInfo } from '../../components/custom-code/code-file'
+import { npmVersion } from '../../components/editor/npm-dependency/npm-dependency'
+import {
+  extractNodeModulesFromPackageResponse,
+  resetDepPackagerCache,
+} from '../es-modules/package-manager/fetch-packages'
+import { mangleNodeModulePaths } from '../es-modules/package-manager/merge-modules'
+import { getJsDelivrFileUrl } from '../es-modules/package-manager/packager-url'
+import * as antdPackagerResponse from '../es-modules/test-cases/antd-packager-response.json'
+import { PackagerServerResponse, requestedNpmDependency } from '../shared/npm-dependency-types'
+import { initPropertyControlsProcessor } from './property-controls-processor'
+import { fullNodeModulesUpdate, getThirdPartyPropertyControls } from './property-controls-utils'
+
+require('jest-fetch-mock').enableMocks()
+
+const simpleCssContent = '.utopiaClass { background-color: red; }'
+
+beforeEach(() => {
```

Comment:
I wonder if we should make this beforeEach a project-level jest configuration

Reply:
Yeah I wonder about this too. The question is where do you draw the line here? We don't want every test performing a load of side effects before each run but at the same point we don't want to have to deal with forgetting to run the specific required side effects in the test files that need them to run. I think with something like this it should be lightweight enough to not matter if we run it before every test :shrug:

Reply:
I think `resetDepPackagerCache` passes the lightweightness test, it sets a variable to `{}`